cypress/e2e/MachineActions.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe("Machine Actions Integration Test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should add and delete machine actions in the gueue", () => {
+    cy.get('[data-testId="action-input"]').type("Action 1");
+    cy.get('[data-testid="add-action-btn"]').click();
+    cy.get('[data-testId="action-input"]').should("have.value", "");
+    cy.contains("Action 1").should("exist");
+
+    cy.get('input[data-testId="action-input"]').type("Action 2");

```

Comment:
It's a duplicate of the previous scenario. Let's consider to have an utils method to reuse this logic for both cases.

Reply:
Also, string values should be set as a const and reuse for both cases. When you are type in this value and when you are checking it. Also **contains** might be quite a risky check, since it's checking the whole page.